docs/docs/faq.md

```diff
@@ -80,6 +80,20 @@ Please note, many keyboards only have a single PCB which includes the “brains
 
 Currently, ZMK only supports wireless split, but wired split is possible and we welcome contributions!
 
+### Pairing Split Keyboards
+
+Only the central half of a keyboard should be paired to a computer or device. The peripheral half will auto-connect to the central half via BLE and should _not_ be paired directly to a computer or device.
```

Comment:
This feels like something that should be outlined in the getting started with ZMK rather than in the FAQ. I doubt many would come here to learn how to pair their device for the first time.

Reply:
Agreed.